docs/tempo/website/configuration/override.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Override trace limit
+weight: 30
+---
+
+# Override trace limit
```

Comment:
This document opens up as if it's about one specific override and then generalizes.  I think a better layout would be:

1) List all 4 current override options.  Include details about what they do and sample log messages when they are tripped.

- ingestion_burst_size
- ingestion_rate_limit
- max_spans_per_trace
- max_traces_per_user

2) Show how to configure these using the standard overrides block:

```
overrides:
    ingestion_rate_limit:
    ...
```
3) Include details about global limit.  Global ingestion rate limit uses the distributor ring to attempt to 

```
overrides:
    ingestion_rate_strategy: global
```

4) Include details about how to do per tenant overrides:

```
overrides:
    per_tenant_override_config: /conf/overrides.yaml
```

file format:

```